### DISCUSSION TITLE
Get Nearest Hostile Mob

### DISCUSSION THREAD

**Discussion by DrMeepso** [PrismarineJS/mineflayer#2222] [General]
I'm trying to get the closest hostile mob to the bot using this code
`bot.nearestEntity(({ kind }) => kind === false)`
but it just returns null

**Comment by WhoTho** [comment]
To get the nearest hostile mob you would use this code (I havn't tested it but believe it should work):
```js
const filter = e => entity.kind === 'Hostile mobs'
const nearestMob = bot.nearestEntity(filter)
```

Hope this helps!
